bonsai/BudgetTransactions/TransactionCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  TransactionCell.swift
+//  bonsai
+//
+//  Created by hoang on 08.12.2021.
+//
+
+import SwiftUI
+
+struct TransactionCell: View {
+    
+    let item: TransactionItem
+    
+    var body: some View {
+        HStack {
+            Image(uiImage: item.image)
```

Comment:
Image should be colored in transaction color

Reply:
fixed

Reply:
👍